README.md

```diff
@@ -1,38 +1,41 @@
 # aimsprop
 
-Read the [documentation](https://mtzgroup.github.io/aimsprop/)!
+To use aimsprop, check out the documentation [here](https://mtzgroup.github.io/aimsprop/)!
 
-## Description
+To develop / contribute to the aimsprop code,
+see [CONTRIBUTING.md](https://github.com/mtzgroup/aimsprop/blob/feature-tutorial-doc/docs/CONTRIBUTING.md) to get your environment and branch set up. 
 
-A repository for the representation and manipulation of AIMS-type trajectories,
-particularly for use in computing time-dependent properties (bond lengths,
-angles, torsions, X-ray scattering, UED, photoelectron, UV-Vis, etc) in a
+For a tutorial on the basic functionality of aimsprop, go [here](https://github.com/mtzgroup/aimsprop/blob/feature-tutorial-doc/docs/tutorial.md).
+
+
+### Description
+
+A repository for the representation and manipulation of AIMS-type trajectories, particularly for use in computing
+time-dependent properties (bond lengths, angles, torsions, X-ray scattering, UED, photoelectron, UV-Vis, etc) in a
 semi-uniform manner.
 
-## Directories
-
-aimsprop - the Python AIMS trajectory/property code
-notes - notes used in implementing
-literature - key papers/book chapters used in implementing certain properties.
-examples - examples of use cases of property computations
-util - utility scripts for common ops like plotting absorption spectra,
-generating difference densities from Molden files, etc (no warrenty on
-these - the usual workflow is to copy these into your own project
-directories and modify until they work)
-
-## Todos
-
-- Population/Excited state lifetime plots (incl. spaghetti)
-- H-Bond coordinates
-- Better UED cross sections
-- OOP angles
-- Update examples
-- Adiabatic dynamics are weirdly done in 0.5 fs, non-adiabatic dynamics are in 20
-  au. These are not the same. We should fix/accomodate this.
-
-## Authors
-
-- Rob Parrish
-- Monika Williams
-- Hayley Weir
-- Colton Hicks
+### Directories
+
+* aimsprop - the Python AIMS trajectory/property code
+* docs - mkdocs documentation that is published [here](https://mtzgroup.github.io/aimsprop/)
+* tests - unit tests for aimsprop code using pytest
+* notes - notes used in implementing
+* literature - key papers/book chapters used in implementing certain properties.
+* scripts - utility scripts for common ops like plotting absorption spectra, generating difference densities from Molden
+  files, etc (no warrenty on these - the usual workflow is to copy these into your own project directories and modify
+  until they work)
+* examples - examples of use cases of property computations (these are not the most helpful, better to look at
+  the [tutorial](https://github.com/mtzgroup/aimsprop/blob/feature-tutorial-doc/docs/tutorial.md)).
+
+### Todos
+
+* H-Bond coordinates
+* Better UED cross sections
+* OOP angles
+* Update examples
+* Adiabatic dynamics are weirdly done in 0.5 fs, non-adiabatic dynamics are in 20 au. These are not the same. We should
```

Comment:
We should be consistent throughout our documentation and use either "-" or "*" for bullets, but not both. The convention is usually to go with "-" as pre the rest of the documentation.